### PR TITLE
fix: resolve remaining errors

### PR DESCRIPTION
**Pull request by samfundev**

### CODE REVIEW

File: src/lib/utils/resolvers/resolveGuildChannelPredicate.ts [hunk 1]
```diff
@@ -12,7 +12,8 @@ export function resolveGuildChannelPredicate<TChannel extends GuildBasedChannelT
 	error: TError
 ): Result<TChannel, TError | Identifiers.ArgumentGuildChannelError> {
 	const result = resolveGuildChannel(parameter, guild);
-	if (!result.isErr()) return result;
-	if (predicate(result.unwrap())) return Result.ok(result.unwrap());
+	if (result.isErr()) return result;
```
Comment (c43721) [inline src/lib/utils/resolvers/resolveGuildChannelPredicate.ts:15]: Can we not do an `unwrapOr` / something of that nature?

File: src/lib/parsers/Args.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ import type {
 	User,
 	VoiceChannel
 } from 'discord.js';
-import type * as Lexure from 'lexure';
+import * as Lexure from 'lexure';
```
Comment (c43721) [inline src/lib/parsers/Args.ts:17]: If possible, can we just plugin `@sapphire/lexure` instead?

**Review by c43721 — CHANGES_REQUESTED** [review]

File: src/lib/utils/resolvers/resolveGuildChannelPredicate.ts (hunk 1 above)
Comment (samfundev) [reply src/lib/utils/resolvers/resolveGuildChannelPredicate.ts:15]: If you're talking about the type guard, there's `isOkAnd` which checks a predicate but doesn't work with type guards. If you're talking about the `isErr` call, I don't get what could be unwrapped.

File: src/lib/parsers/Args.ts (hunk 2 above)
Comment (samfundev) [reply src/lib/parsers/Args.ts:17]: I was going to do that in a separate PR, but I can do that in this one.

Comment (c43721) [reply src/lib/parsers/Args.ts:17]: please do if possible! That'd be excellent, but up to you.

File: src/lib/utils/resolvers/resolveGuildChannelPredicate.ts (hunk 1 above)
Comment (c43721) [reply src/lib/utils/resolvers/resolveGuildChannelPredicate.ts:15]: No worries then. This is good enough.

**Comment by c43721** [comment]
thanks!

**Pull request merged**